public/css/screen.styl

```diff
@@ -203,6 +203,189 @@ a.dropdown-toggle {
 .event-navbar .admin-button {
   margin-top: 15px;
 }
+/* ================== EMAIL PREVIEW MODE STYLES ==================== */
+
+.email-preview-container {
+  padding: 0px;
+  margin-top: 0px;  
+  font-family: Helvetica,Arial,sans-serif;
+
+  .col-lg-12 {
+    padding: 0px;
+  }
```

Comment:
Please do not alter core bootstrap styles for particular instances where something needs to be different.  It will be confusing for other developers who are familiar with bootstrap and expect a certain behavior from ``col-lg-12`` and get something else.

Instead, either replace ``col-lg-12`` with your own class that implements just the parts of ``col-lg-12`` that you want, or add a second class (perhaps ``no-padding``) which you use in addition to ``col-lg-12``.  Either way, someone reading the html should be able to guess what's going on.

This applies to many modifications below, including: ``control-label``, ``form-group``, other column classes, etc.